spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestViews.java

```diff
@@ -877,6 +877,9 @@ public void dropV1View() {
             v1SessionCatalog()
                 .tableExists(new org.apache.spark.sql.catalyst.TableIdentifier(v1View)))
         .isFalse();
+
+    sql("USE spark_catalog");
+    sql("DROP TABLE IF EXISTS %s", tableName);
```

Comment:
the existence of that table in `spark_catalog` would not cause the expected failures in the newly added tests when doing a read against the table